SwiftWisdom/Core/UIKit/NibLoadableView.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  NibLoadableView.swift
+//  SwiftWisdom
+//
+//  Created by Christian Howland on 3/31/17.
+//  Copyright © 2017 Intrepid. All rights reserved.
+//
+
+import UIKit
+//
+// Subclassing NibLoadableView allows you to automate the loading of a Nib into a placeholder view inside another Nib, using only the Custom Class attribute.
+// Specifically, it removes the need to perform the following manual steps every time you load a Nib into a placeholder view:
+//   ->  setting 'translatesAutoresizingMaskIntoConstraints' to false
+//   ->  adding constraints to fit the new view into its container
+//   ->  writing boilerplate Nib loading code.
+//
+// EXAMPLE: if you have an outer Nib with a placeholder view inside it having custom class "MyView", you would, at a
+// minimum, just need to implement the following code:
+//      " class MyView: NibLoadableView {} "
+// Then, as soon as the outer Nib is loaded and the 'MyView' Custom Class attribute is seen, that view will be instantiated,
+// loaded into the placeholder view, and constrained to each edge of the containing view.
+//
+// You should only subclass NibLoadedView if you want the ability to load the view into a placeholder within a Nib, i.e., with with Custom Class & initWithCoder.
```

Comment:
Can you make this description into quick help? (i.e. add another `/` to each line)